### DISCUSSION TITLE
Why is the tab title is so long in Linux?

### DISCUSSION THREAD

**Discussion by ChenZhongPu** [kovidgoyal/kitty#6175] [Q&A]
I installed kitty on both Linux and macOS with the default settings, but the style of tab title varies.

In macOS, the tab title is expected: either the `path` or the `current shell command`:

<img width="403" alt="image" src="https://user-images.githubusercontent.com/2310958/231155459-743477c2-c25c-4d78-816e-8ecf07f07f42.png">

However, its title in Linux is very long, and it is prefixed with `username@device-name`:

<img width="1022" alt="image" src="https://user-images.githubusercontent.com/2310958/231155807-49da3266-7cea-4b95-8bf9-0cefe447db4c.png">

Why does they have different styles? And how to make them consistent?

**Comment by kovidgoyal** [comment]
Presumably in your linux box you have some shell config that causes it
to send that title to the terminal emulator. Remove that and you will
have the same behavior as macOS.

**Reply by ChenZhongPu** [reply]
Do you know where the possible `shell config` comes from?

**Comment by kovidgoyal** [comment]
man whatever-your-shell-is

**Reply by ChenZhongPu** [reply]
Thanks. The following works for me:

```sh
ZSH_THEME_TERM_TITLE_IDLE="%~"
```
